src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -254,4 +256,11 @@ private function applyCheckoutStateTransition(OrderInterface $order, string $tra
     {
         $this->stateMachineFactory->get($order, OrderCheckoutTransitions::GRAPH)->apply($transition);
     }
+
+    private function setCompleteCheckoutDate(OrderInterface $order): void
+    {
+        if ($order->getCheckoutState() === OrderCheckoutStates::STATE_COMPLETED) {
+            $order->setCheckoutCompletedAt($this->faker->dateTimeBetween('-1 years', 'now'));
```

Comment:
Could we add this line to the end of `completeCheckout` method? We will not need this `if` then and it will be more concise (completed checkout date is directly related to completing checkout).